server/src/tenant/custom/args/custom-entity-order-by-relation.input.ts

```diff
@@ -0,0 +1,29 @@
+import { Field, InputType } from '@nestjs/graphql';
```

Comment:
@magrinj we should not use the term "custom" here as we will query both standards, remote, and custom objects. I think tenant is a good term, I would not introduce an additional custom nesting!